packages/react-native-media-player/ios/MediaPlayer.swift

```diff
@@ -0,0 +1,29 @@
+import AVFoundation
+
+@objc(MediaPlayer)
+public class MediaPlayer: NSObject {
+
+    var avPlayer:AVAudioPlayer!
+  @objc(load:resolver:rejecter:)
+  public func load(to url: String, resolve: RCTPromiseResolveBlock, reject: RCTPromiseRejectBlock) {
+    do {
+        let fileURL = URL(string:url)
+        let soundData = try Data(contentsOf:fileURL!)
```

Comment:
Codacy found an issue: [Colons should be next to the identifier when specifying a type and next to the key in dictionary literals.](https://app.codacy.com/manual/yajananrao/Nadha/pullRequest?prid=5728824)